OasisDEX/oasis-borrow
# Added ATP summary after execution

# [Added ATP summary after execution](https://app.shortcut.com/oazo-apps/story/6321/ui-auto-take-profit-summary-after-execution)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added summary banner and form to overview when ATP has been executed (in the same manner as for SL)
- removed bunch of unused components
- extracted some common components
  
## How to test 🧪
  <Please explain how to test your changes>

- visit vault 345 on goerli to see it, (407 SL executed for comparison), you should be able to close banner and reopen position
